### PR TITLE
Test coverage for default gas preferences 

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
Test coverage for #13238

### CODE REVIEW

File: ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js
Comment (jpuri) [inline ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js:58]: It will be better to have `...storeProps,` towards end.

File: ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js
Comment (jpuri) [inline ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js:202]: Here would be nice to have test to check that default advance fee is visible even if userFeeLevel is not custom.

**Review by danjm — APPROVED** [review]
I think this is good to go. Will wait for @jpuri's approval

File: ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js
Comment (jpuri) [inline ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js:191]: Test cases look redundant, it will be nicer to test that advance defaults are actually visible on edit gas fee modal.

**Comment by jpuri** [comment]
Hey @NiranjanaBinoy : added comment above. That needs to be addressed before PR can be merged.

**Pull request closed without merging**
